README.md

```
@@ -281,8 +280,11 @@
 - *"Create MCP streaming proxy for ai-service at ai.example.com:8080"*
 
 **Key Options:**
-- **Config Types**: subdomain, subfolder, mcp-subdomain, mcp-subfolder
-- **Authentication**: none, ldap, authelia, authentik, tinyauth (default: authelia)
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)